charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -259,6 +259,8 @@ spec:
             value: "kube_endpoints kube_services"
           - name: DD_EXTRA_LISTENERS
             value: "kube_endpoints kube_services"
+          - name: DD_SECRET_BACKEND_SKIP_CHECKS
+            value: "true"
```

Comment:
I do wonder if you are not using the Cluster Checks agent, and want your secrets decrypted on the Cluster Agent, will this affect that?

Reply:
I was about to leave a note. Can you remove this part entirely? 
It will break checks containing ENC that are supposed to be run by the Cluster Agent.
I will make a subsequent PR to add this with a flag and document it.
For now you can use:
```
clusterAgent
  env:
      - name: DD_SECRET_BACKEND_SKIP_CHECKS
        value: true
```

Reply:
removed